### ISSUE TITLE
Support the `set` command.

### ISSUE THREAD

**Issue by skysch** [skysch/atma-rs#5]
Currently unimplemented. It should allow setting Settings options, cursor position, and history state. If a Settings file doesn't exist, one should be created automatically if the Config allows it, or else an erro should be emitted.

- [X]    atma set name POSITION_SELECTOR [NAME]
- [X]    atma set group SELECTION [NAME] [--remove]
- [X]    atma set expr CELL_REF INSERT_EXPR
- [X]    atma set cursor [POSITION]
- [X]    atma set history [enable|disable|clear]
- [X]    atma set active-palette [PATH]
- [X]    atma set delete-cursor-behavior CURSOR_BEHAVIOR
- [X]    atma set insert-cursor-behavior CURSOR_BEHAVIOR
- [X]    atma set move-cursor-behavior CURSOR_BEHAVIOR

**Comment by skysch** [comment]
```
    atma set name [name] POSITION_SELECTOR..
    atma set expr [SetExpr] CELL_REF..
    atma set active_palette [path]
    atma set cursor [Position]
    atma set history [enable|disable|clear]
    atma set delete_cursor_positioning CURSOR_POSITIONING
    atma set insert_cursor_positioning CURSOR_POSITIONING
    atma set move_cursor_positioning CURSOR_POSITIONING
```

**Comment by skysch** [comment]
Fixed in 0.0.176.
